cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/dichotomy/MultipleDichotomyRunner.java

```diff
@@ -50,14 +63,30 @@ public MultipleDichotomyResult<DichotomyRaoResponse> runMultipleDichotomy(CseReq
         forcedPrasIds.add(new HashSet<>(request.getManualForcedPrasIds()));
 
         if (businessLogger.isInfoEnabled()) {
-            businessLogger.info("Multiple dichotomy runs: Initial dichotomy running, Forcing following PRAs: {}", printablePrasIds(forcedPrasIds));
+            businessLogger.info("Multiple dichotomy runs: Initial dichotomy running, Forcing following PRAs: {}", toString(flattenPrasIds(forcedPrasIds)));
         }
 
         // Launch initial dichotomy and store result
         DichotomyResult<DichotomyRaoResponse> initialDichotomyResult =
             dichotomyRunner.runDichotomy(request, cseData, network, initialItalianImport, flattenPrasIds(forcedPrasIds));
         multipleDichotomyResult.addResult(initialDichotomyResult, flattenPrasIds(forcedPrasIds));
 
+        String printablePrasIdsIds = "";
+        if (flattenPrasIds(forcedPrasIds) != null) {
+            printablePrasIdsIds = toString(flattenPrasIds(forcedPrasIds));
+        }
+
+        businessLogger.info(SUMMARY, "First run",
+            round(dichotomyResultHelper.computeHighestSecureItalianImport(initialDichotomyResult)),
+            TtcResult.limitingCauseToString(initialDichotomyResult.getLimitingCause()),
+            dichotomyResultHelper.getLimitingElement(initialDichotomyResult),
+            printablePrasIdsIds,
+            "NONE");
```

Comment:
I think you inverted forced PRAs and PRAs, and it won't be none you have to retrieve them from the dichotomy result